### PR TITLE
feat: add pprof example

### PR DESCRIPTION
**Pull request by wangbei98**
#### What type of PR is this?

feat: add pprof example

#### What this PR does / why we need it (English/Chinese):

en: add pprof example
zh: 增加pprof示例代码

### CODE REVIEW

**Comment by li-jin-gou** [comment]
refer to https://github.com/cloudwego/hertz-examples/pull/36

File: go.mod [hunk 1]
```diff
@@ -11,7 +11,7 @@ require (
 	github.com/bytedance/gopkg v0.0.0-20220824043955-beb90005fda6 // indirect
```
Comment (Skyenought) [inline go.mod:11]: go mod tidy

File: middleware/pprof/README.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+## How to run
```
Comment (Skyenought) [inline middleware/pprof/README.md:1]: 没有在根目录添加导航

File: go.mod (hunk 1 above)
Comment (wangbei98) [reply go.mod]: GET

File: middleware/pprof/README.md (hunk 2 above)
Comment (wangbei98) [reply middleware/pprof/README.md]: GET

**Comment by li-jin-gou** [comment]
LGTM

**Review by li-jin-gou — APPROVED** [review]

**Pull request merged**